elliott/elliottlib/runtime.py

```diff
@@ -277,37 +286,8 @@ def initialize_logging(self):
         else:
             log_level = logging.INFO
 
-        default_log_formatter = logging.Formatter('%(asctime)s %(levelname)s %(message)s')
-
-        root_logger = logging.getLogger()
-        root_logger.setLevel(logging.WARN)
-        root_stream_handler = logging.StreamHandler()
-        root_stream_handler.setFormatter(default_log_formatter)
-        root_logger.addHandler(root_stream_handler)
-
-        # If in debug mode, let all modules log
-        if not self.debug:
-            # Otherwise, only allow children of ocp to log
-            root_logger.addFilter(logging.Filter("ocp"))
-
-        # Get a reference to the logger for elliott
-        self.logger = logutil.get_logger()
-        self.logger.propagate = False
-
-        # levels will be set at the handler level. Make sure main level is low.
-        self.logger.setLevel(logging.DEBUG)
-
-        main_stream_handler = logging.StreamHandler()
-        main_stream_handler.setFormatter(default_log_formatter)
-        main_stream_handler.setLevel(log_level)
-        self.logger.addHandler(main_stream_handler)
-
-        self.debug_log_path = os.path.join(self.working_dir, "debug.log")
-        debug_log_handler = logging.FileHandler(self.debug_log_path)
-        # Add thread information for debug log
-        debug_log_handler.setFormatter(logging.Formatter('%(asctime)s %(levelname)s (%(thread)d) %(message)s'))
-        debug_log_handler.setLevel(logging.DEBUG)
-        self.logger.addHandler(debug_log_handler)
+        logutil.init_logging(log_level, self.debug_log_path)
```

Comment:
same as commented above for doozer, where is `init_logging` defined?

Reply:
I renamed it to setup_logging.